### ISSUE TITLE
ENH: Add tests and CI

### ISSUE THREAD

**Issue by csmcallister** [DataKind-DC/capital-nature-ingest#13]
We should write some unit tests for our lambda functions and set up CI and logging. This will help let us know where/how things break once they inevitably do.

## Expected Behavior
 - [x] Each lambda function has its own test suite with decent code coverage.
 - [x] CI is set up.
 - [ ] Logging is set up.

## Current Behavior
We don't have any tests, CI or logging 😭 

## Possible Solution
Use [httpretty](https://pypi.org/project/httpretty/) to mock responses for the sites we scrape and the APIs we hit. We can hardcode mock responses as Python dicts (for the APIs that return JSON) and as binary strings for the html content that we scrape. Tedious, yet. But it'll save time debugging issues in the future. 

## Context
This isn't super important as we can manually cobble together a csv for Wordpress to ingest, but writing some tests and setting up CI will help us in the long-term when it comes to maintaining this contributors come and go.

**Comment by csmcallister** [comment]
@abarciauskas-bgse Is it possible for DataKind to create a free CircleCI project for this repo? I tried, but then I realized I don't have access to the repo since it's under the DataKind-DC GitHub org. This isn't super important right now, but it would be handy for checking the event schemas people contribute and catching unintended breaking changes. Once the project is set up, I can add the config.
